pkg/components/exporter/exporter.go

```diff
@@ -121,6 +121,7 @@ func NewDaemonSet(detail components.Detail, k *v1alpha1.Kepler) *appsv1.DaemonSe
 					Labels:    podSelector,
 				},
 				Spec: corev1.PodSpec{
+					HostPID:            true,
```

Comment:
Is this required?

Reply:
Hey @sthaha, I added this because kepler mounts /proc and therefore can access the PID namespace of the host. I am not sure about the internals of kepler yet but apparently it does not interfere with the host PID namespace yet. However, if we don't set `HostPID: true`, the kepler pod will have its own PID namespace. In this kepler PID namespace maybe some process for example has PID 0 or 1 and another process on the host could have the same PID. If in the future kepler would somehow interact with this PID there could be collisions and unwanted behavior like interacting with systemd without knowing. To prevent this, `HostPID: true` should be set so there are no PID collisions between host and kepler pod.

Reply:
@rootfs may be `kepler` manifests should have this as well